HW2/Task2.py

```diff
@@ -0,0 +1,58 @@
+# Напишите программу, которая принимает две строки вида “a/b” - дробь с числителем и знаменателем.
+# Программа должна возвращать сумму и произведение* дробей.
+# Для проверки своего кода используйте модуль fractions.
+import fractions
+
+a = '324234/2345'
+b = '324/2342'
```

Comment:
Указал такие дроби
```suggestion
a = '0/2'
b = '1/2'
```
И вот что получил
![image](https://github.com/Zelendewagen/DPY/assets/49741194/7a7fabe9-9628-49c2-837d-61e90b3ab8cb)